config/jobs/kubernetes/sig-node/node-kubelet.yaml

```
@@ -408,7 +408,7 @@ periodics:
           - --node-test-args=--kubelet-flags="--exit-on-lock-contention --lock-file=/var/run/kubelet.lock"
           - --node-tests=true
           - --provider=gce
-          - --test_args=--nodes=1 --focus="\[NodeFeature:LockContention\]" --restart-kubelet=false
+          - --test_args=--nodes=1 --focus="\[NodeFeature:LockContention\]" --skip="\[Flaky\]|\[Serial\]"
```

why do we need this new skip flag? it will not do much, will it?

---

@SergeyKanzhelev Locally If I don't add a `skip`, the test is getting skipped.  Locally I had to add `SKIP=""`

---

oh, I see. You just override the default. In this case you might try to override it to empty.

---

```suggestion
          - --test_args=--nodes=1 --focus="\[NodeFeature:LockContention\]" --skip=""
```

like this ?

@SergeyKanzhelev

---

yes, this expresses the intent better I think. As long as it works

---

done.